tests/layer_tests/ovc_python_api_tests/test_ovc_cli_tool.py

```diff
@@ -155,3 +157,16 @@ def test_ovc_tool_saved_model_dir_with_sep_at_path_end(self, ie_device, precisio
         ov_model = core.read_model(os.path.join(temp_dir, "test_model.xml"))
         flag, msg = compare_functions(ov_model, ref_model, False)
         assert flag, msg
+
+    def test_ovc_tool_non_existng_output_dir(self, ie_device, precision, ir_version, temp_dir, use_new_frontend, use_old_api):
```

Comment:
can we remove `use_new_frontend` because this option is not needed for `ovc`? And other options like: `use_old_api`, `precision` (instead better to use `compress_to_fp16`)

Reply:
Done for this set of tests.